### ISSUE TITLE
 I'm New to this domain

### ISSUE THREAD

**Issue by Rickustc** [koncle/DomainAdaptor#2]
Great job! I'm not very familiar with testing-time domain adaptation. Could you please clarify if it can be used to enhance the performance of a pre-trained model in downstream zero-shot inference applications? For example, improving the annotation performance of a pre-trained object detector without corresponding labels in the target domain data. Thank you! ：D

**Comment by koncle** [comment]
Sorry for the late reply and thank you for your interest! Although we only performed experiments in the classification task, we believe that it is also applicable to other tasks. And the performance improvement may be dependent on the specific tasks. 
Note that 1) since the AdaMixBN depends on the batch normalization layer, the pre-trained model should also contain this module; 2) For a better adaptation to the unlabeled batch, the base performance should be relatively good, such that it can learn from the correctly predicted data.

**Comment by Rickustc** [comment]
Thankyou for your reply！
